gitpod-io/website
# ✨ Revamp `/community` page

## Description


Minor revamp of `/community` page

## Related Issue(s)



Fixes #3700

## How to test

- Open preview
- Go to `/community` page & see 👀


## Release QA Checklist ✔️



<details>
  <summary>QA Checklist </summary>

## [Priority: High] Functional Testing

  

## [Priority: High] Content Testing


## [Priority: Medium] Performance Testing


## [Priority: Medium] Accessibility Testing


By including these key items in your website release QA checklist, you can ensure that your website is fully functional, user-friendly, and meets your business requirements.

## Security Testing

> **Note**:💡 If we are changing something related to cookies or data storing or analytics side.


  </details>


<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3703"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
